### PR TITLE
[Retribution Paladin] Refactoring

### PR DESCRIPTION
**Pull request by CollCrom**
Mostly changed a bunch of spans to Wrappers. I fixed a few suggestion tooltips and also removed some duplicate spells from the ABC module.

### CODE REVIEW

File: src/Parser/Core/Modules/Haste.js [hunk 1]
```diff
@@ -41,7 +41,6 @@ class Haste extends Analyzer {
     [209166]: 0.3, // DEBUFF - Fast Time from Elisande
     [209165]: -0.3, // DEBUFF - Slow Time from Elisande
     // [208944]: -Infinity, // DEBUFF - Time Stop from Elisande
-    [SPELLS.BONE_SHIELD.id]: 0.1, // Blood BK haste buff from maintaining boneshield
```
Comment (MartijnHols) [inline src/Parser/Core/Modules/Haste.js]: Why did you remove this?

Comment (CollCrom) [reply src/Parser/Core/Modules/Haste.js]: It was repeated. I don't have line numbers on my phone but it's right below Icy Veins
